### ISSUE TITLE
svelte-parser cannot parse code if `{/if}` contains whitespace

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#1321]
## Describe the bug

Svelte allows whitespaces or line breaks in logic blocks.

```svelte
{
	#if
	true
}
	<p>true</p>
{
	:else
}
	<p>false</p>
{
	/if
}
```

I checked it on official REPL environment, and there is no errors
https://svelte.jp/repl/hello-world?version=4.2.8
![image](https://github.com/markuplint/markuplint/assets/34136752/54e36ed6-61e7-4a71-a141-109774e2f016)

but svelte-parser fails to parse

https://github.com/markuplint/markuplint/blob/69862cbe7d148afaf9a9cfe94e4e3fa6009252c2/packages/%40markuplint/svelte-parser/src/parse-ctrl-block.ts#L200-L203
